docs/pbc/all/customer-relationship-management/202212.0/import-and-export-data/file-details-company-business-unit.csv.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "File details: company_business_unit.csv"
+description: Use the company_business_unit.csv file to configure information about company accounts in your Spryker shop.
+template: data-import-template
+last_updated: Jun 1, 2023
+related:
+  - title: Execution order of data importers in Demo Shop
+    link: docs/scos/dev/data-import/page.version/demo-shop-data-import/execution-order-of-data-importers-in-demo-shop.html
+---
+
+This document describes the `company_business_unit.csv` file to configure information about [business units](docs/pbc/all/customer-relationship-management/{{page.version}}/company-account-feature-overview/business-units-overview.html) in your Spryker shop.
+
+## Import file dependencies
+
+[TODO: Add dependencies if any; remove the section otherwise]
+
+## Import file parameters
+
+| PARAMETER | REQUIRED |  TYPE | DESCRIPTION |
```

Comment:
Review the table; make sure that the REQUIRED, TYPE, AND DESCRIPTION fields contain correct information.